src/aspire/nfft/cufinufft.py

```diff
@@ -124,7 +124,7 @@ def default_opts(finufft_type, dim):
     c_int, c_float, c_int, CufinufftPlan_p]
 _make_plan.restypes = c_int
 
-def plan(finufft_type, modes, isign, tol, opts=None):
+def plan(finufft_type, modes, isign, tol, many=1, opts=None):
```

Comment:
Should the value of the many variable be int or bool type as the self.many in line 190?

Reply:
This plan belongs to cufinufft, and the arguments have different meaning from the Plan class that belongs to aspire. It should probably be an integer in this case.

These is some ambiguity between a single case and a many batch of size 1. That is why there is None logic in the other location.

Reply:
To be clear, the bindings are not part of aspire, they are just temporarily here for the hackathon. The parent library is changing to API a little right now...